src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,178 @@
+package duke;
+
+import java.io.*;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class that creates Storage object to do file manipulation
+ * @author amresh A0235398R
+ */
+public class Storage {
+
+    protected String filePath;
+    protected ArrayList<Task> storeLists = new ArrayList<>();
+
+    /**
+     * Constructor that creates Storage object
+     * @param filePath Location where file is stored
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Loads file at the beginning of the program
+     * @return ArrayList<Task> that will return when method runs
+     */
+    public ArrayList<Task> loadFile() {
+        try {
+            File myObj = new File(filePath);
+            if (myObj.createNewFile()) {
+                System.out.println("No tasks at the moment");
+            } else {
+                try {
+                    printFileContents(filePath);
+                } catch (FileNotFoundException e) {
+                    System.out.println("File not found");
+                }
+            }
+        } catch (IOException e) {
+            System.out.println("An error occurred.");
+            e.printStackTrace();
+        }
+        return storeLists;
+    }
+
+    /**
+     * Prints File Contents onto console
+     * @param filePath Location where file is stored
+     * @throws FileNotFoundException Throws exception if file is not available
+     */
+    public void printFileContents(String filePath) throws FileNotFoundException {
+        try (BufferedReader br = new BufferedReader(new FileReader(filePath))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                renderStringsAsTasks(line);
+            }
+        } catch (IOException ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    /**
+     * Method that parses strings and converts them to Tasks
+     * @param taskStr String that will be passed into method to convert
+     */
+    public void renderStringsAsTasks(String taskStr) {
```

Comment:
A bit of a nitpick but perhaps this method could be renamed? `render` usually has other meanings (I'm thinking 3D render etc). I like your description in the accompanying javadocs, so perhaps `convertStringToTask` would be a more intuitive name? I've removed the plurality as well (`Strings` to `String`) as the method operates on one String and returns one Task. 
```suggestion
    public void convertStringToTask(String taskStr) {
```